vspace/src/main/java/edu/asu/diging/vspace/web/staff/DeleteSpacesCustomOrderController.java

```diff
@@ -0,0 +1,30 @@
+package edu.asu.diging.vspace.web.staff;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import edu.asu.diging.vspace.core.services.ISpacesCustomOrderManager;
+
+@Controller
+public class DeleteSpacesCustomOrderController {
+    
+    @Autowired
+    private ISpacesCustomOrderManager spacesCustomOrderManager;
+    
+    @RequestMapping(value = "/staff/space/customorder/{orderId}", method = RequestMethod.POST)
```

Comment:
request should be `DELETE` if this is the url. The url suggests that a post to it will update the object not delete it.